core/dbt/task/deps.py

```diff
@@ -23,11 +23,17 @@
 from dbt.task.base import BaseTask, move_to_nearest_project_dir
 
 
+from dbt.config import RuntimeConfig, Project
+from dbt.task.base import NoneConfig
+
 class DepsTask(BaseTask):
```

Comment:
Since we are doing the refactor here, does this still have to be a Task object? Seems like we are not getting much stuff from the `BaseTask'?

Reply:
There may be a more appropriate base class for more util-like commands such as clean, deps, init and debug. At the risk of abstracting too early I'm thinking to leave this line as-is for this PR and revisit it once we've got clean/init/debug combed through - there's probably a common set of functionality that makes sense for util-type commands that can be separate from BaseTask.